src/js/collapsible-text.jsx

```diff
@@ -0,0 +1,59 @@
+var React = require('react'),
+    _ = require('underscore'),
+    marked = require('marked');
+
+marked.setOptions({
+    renderer: new marked.Renderer(),
+    gfm: true,
+    tables: true,
+    breaks: false,
+    pedantic: false,
+    // sanitize has to be false, otherwise embedded html isn't rendered correctly
+    sanitize: false,
+    smartLists: true,
+    smartypants: false
+});
+
+var Collapsible = React.createClass({
+    propTypes: {
+        title: React.PropTypes.string.isRequired,
+        content: React.PropTypes.string,
+    },
+
+    getDefaultProps: function() {
+        return {
+            content: '',
+        };
+    },
+
+    getInitialState: function() {
+        return {
+            isCollapsed: false,
+        };
+    },
+
+
+    render: function() {
+        var contents = this.state.isCollapsed ? null: 
+            <div className='paragraph' dangerouslySetInnerHTML={{__html: marked(this.props.content)}} />
+
+        return(<div className='panel-group'>
```

Comment:
For multi-line HTML, let's keep the convention that the opening paren is on the same line as the `return` and the markup starts on the next line.

```jsx
return (
    <div ... />
);
```

Update all instances of this. In addition ensure there is always a space preceding a left paren, `return (` rather than `return(` or `if(`.